src/Google.Cloud.Diagnostics.Debug/Google.Cloud.Diagnostics.Debug/LoggingClient.cs

```diff
@@ -66,10 +67,10 @@ public WriteLogEntriesResponse WriteLogEntry(StackdriverBreakpoint breakpoint)
         /// Substitutes the $0, $1, etc. in messageFormat with expressions
         /// from evaluatedExpressions.
         /// </summary>
-        /// <returns></returns>
+        /// <returns>Formatted log message with expressions substituted.</returns>
         private string SubstituteLogMessageFormat(string messageFormat, List<Debugger.V2.Variable> evaluatedExpressions)
         {
-            string result = "";
+            string result = "LOGPOINT: ";
```

Comment:
Is this standard to prefix it this way?

Reply:
When you click "View logs" button on the Stackdriver UI, it queries for logs that starts with "LOGPOINT: "